### PR TITLE
Add SHOW GLOBAL VARIABLES metris

### PR DESCRIPTION
**Pull request by SuperQ**
SHOW GLOBAL VARIABLES provides gauge-like views of various MySQL
internal tunable settings.  These can be used for expressions that
compare current status (mysql_global_status_connections /
mysql_global_variables_max_connections)

### CODE REVIEW

**Comment by SuperQ** [comment]
@roman-vynar This will give us all the metrics for `SHOW GLOBAL VARIABLES`.

**Comment by roman-vynar** [comment]
That's great :+1:

File: mysqld_exporter.go [hunk 1]
```diff
@@ -302,6 +303,34 @@ func (e *Exporter) scrape(ch chan<- prometheus.Metric) {
...
+		e.error.Set(1)
+		return
+	}
+	defer globalVariablesRows.Close()
+
+	var globalVarKey string
+	var globalVarValue sql.RawBytes
+
+	for globalVariablesRows.Next() {
+		globalVarKey = strings.ToLower(globalVarKey)
```
Comment (matthiasr) [inline mysqld_exporter.go]: Why is this before the `Scan`? Seems like a NOOP here.

**Comment by matthiasr** [comment]
:+1: when the toLower is in the right spot

**Comment by SuperQ** [comment]
@matthiasr Ahh, right, had it in the wrong place.

**Pull request merged**